### ISSUE TITLE
How can I create a ViewFactory in v0.32?

### ISSUE THREAD

**Issue by LeoReubelt** [ivmartel/dwv#1425]
I have a function to that uses ViewFactory.

i used to do the following...
```js
const dwv = require(`dwv`);
const viewFactory = new dwv.image.ViewFactory();
```
After upgrading to 0.32, I get the following...
TypeError: Cannot read properties of undefined (reading 'ViewFactory')

And I can't seem to figure out how to get it now.

**Comment by ivmartel** [comment]
Instead of creating an instance of a factory, you now can just call `createView(elements, image)` to create a `View` (see [createView](https://ivmartel.github.io/dwv/doc/stable/global.html#createView) in the doc).

**Comment by LeoReubelt** [comment]
Thanks @ivmartel.

**Comment by LeoReubelt** [comment]
@ivmartel - do i have to be on 0.32.1 for this to work? Because i still can't seem to get it to work. I can not seem to import it.

**Comment by ivmartel** [comment]
Yes, it's on 0.32.1 (#1410).

**Comment by LeoReubelt** [comment]
ok. thanks.

**Comment by ivmartel** [comment]
In what context? Are you trying to load a data?

**Comment by ivmartel** [comment]
This error is when the app tries to use an already assigned index. Are you loading with `loadURLs`?

**Comment by LeoReubelt** [comment]
no, i am using App.loadFiles. we get zip files that sometimes contain multiple series mixed together, subfolders, non-dicom files, and rtstruct or rtplan modalities. so first i open the zip, separate everything by file type, series, and modality. and then i use App.loadFiles and pass it files: File[].

**Comment by LeoReubelt** [comment]
but if if i make urls from the files and use loadURLs, i get the same errors.

```
      const urls = props.files.map((file) => URL.createObjectURL(file));
      app.loadURLs(urls);
```

**Comment by ivmartel** [comment]
Did you find a solution to your problem?

**Comment by LeoReubelt** [comment]
@ivmartel yes thanks. i don't actually know what fixed it. otherwise i would have put it here. but i had a remaining require('dwv'). and when i changed it to import some of the functions changed. so i am assuming that is what fixed it.

**Comment by ivmartel** [comment]
dwv 0.32 allows to do individual imports such as `import {App} from 'dwv'`. I would recommend it if your environment supports it instead of a global import or require. That way you know exactly what you depend upon.

**Comment by git-toni** [comment]
Hello @LeoReubelt , sorry to ping you in here 1 year later :pray: 
Once you have the new view created, how do you "append" it to the already initialized `dwv.App`  instance?
I'm missing some sort of
```javascript
app.init(...)
//....
const newView  = dwv.createView(elements, image)
app.appendNewView(newView) // would create a new dataId layerGroup including the new view
app.render() // to refresh the rendering given the new View
```
What am I missing?
